iree/compiler/Conversion/Common/LinalgBufferizePass.cpp

```diff
@@ -231,6 +231,21 @@ static LogicalResult convertConstantOp(OpBuilder &b, ConstantOp constantOp,
   return success();
 }
 
+/// Converts a linalg.init_tensor op to memref.alloc op. This provides a shaped
+/// operand for pooling ops. The op will be deleted after going to loops.
+static LogicalResult convertInitTensorOp(OpBuilder &b,
+                                         linalg::InitTensorOp initTensorOp,
+                                         BlockAndValueMapping &bvm) {
+  RankedTensorType tensorType = initTensorOp.getType();
```

Comment:
You probably need to check if the result is already mapped to a buffer here. There is a pre-processing step for Linalg operations that tries to map the result buffer (if it is already mapped) to the `outs` operand. So here this will be undone.

Reply:
I see, thanks!

Reply:
Hey Hanhan, this is probably needed before this can land.

Reply:
Does `if (bvm.contains(initTensorOp.getResult())) return success();` address the comment? I thought `bvm` contains the information of mapping between `outs` and buffers.

Reply:
Right. That works. Missed that.